site3/website/docusaurus.config.js

```diff
@@ -7,7 +7,7 @@ const baseUrl = process.env.BASE_URL || "/"
 const deployUrl = process.env.DEPLOY_URL || "https://bookkeeper.apache.org";
 const variables = {
   /** They are used in .md files*/
-  latest_release: "4.16.3",
```

Comment:
not sure about this, I think "latest" is intended as the release with the latest features (so 4.16 will always be ahead)

Reply:
okay. make sense. I thought it is the newest release.